src/main/scala/org/ergoplatform/settings/Parameters.scala

```diff
@@ -0,0 +1,14 @@
+package org.ergoplatform.settings
+
+object Parameters {
```

Comment:
Some class comments might be useful here. I expect it is assumed to be class, and it's parameters should be changed by miner votes?

Reply:
yes, a comment added